client/src/reducers/web3.ts

```diff
@@ -67,6 +69,13 @@ export const web3Reducer = (
         account: action.account,
       };
     }
+
+    case WEB3_BAD_CHAIN_ERROR: {
```

Comment:
if we use the error type like commented above I think we can skip this case since it's already checked with `WEB3_ERROR`